### ISSUE TITLE
Auto install babel presets

### ISSUE THREAD

**Issue by youknowriad** [remixz/run-js#9]
Hi, and tks for this great tool,

Since the lib is already using babel why not enabling parsing the .babelrc and auto installing the presets and plugins ?

Thanks

**Comment by martinschayna** [comment]
I agree, it will be better. Now it is possible to use babel presets in run-js by manually installing them via `npm install`. Anyway, great tool!
